src/components/Column/Column.tsx

```diff
@@ -0,0 +1,49 @@
+import { CSSProperties, useState } from "react";
+import styled from "styled-components";
+
+import Card from "../Card";
+import Ticket from "../../interfaces/Ticket";
+import { ColumnHeader } from "./ColumnHeader";
+import { ColumnFooter } from "./ColumnFooter";
+import { ColumnBody, DraggableColumnBody } from "./ColumnBody";
+
+export interface ColumnProps {
+  title: string;
+  items?: Ticket[];
+  draggable?: boolean;
+  onEditClick?: () => void;
+  onAddTicket?: () => void;
+  className?: string;
+  style?: CSSProperties;
+}
+
+export default function Column({
+  title,
+  items = [],
+  draggable,
+  onEditClick,
+  onAddTicket,
+  className,
+  style,
+}: ColumnProps): JSX.Element {
+  const [tickets, setTickets] = useState<Ticket[]>(items);
+
+  return (
+    <Body className={className} style={style}>
+      <ColumnHeader title={title} />
+      {draggable ? (
+        <DraggableColumnBody setTickets={setTickets} title={title} tickets={tickets} onEditClick={onEditClick} />
+      ) : (
+        <ColumnBody title={title} tickets={tickets} onEditClick={onEditClick} />
+      )}
+      <ColumnFooter onAddTicket={onAddTicket} />
+    </Body>
+  );
+}
```

Comment:
`ColumnHeader` and `ColumnFooter` could stay in this file too, I won't mind bringing them back. But I liked this way since this `Column` component is just a parent of these three parts. Which are the header, body, and footer. It really comes to preference in the end though, I also see some people prefer to make small components coupled together.